samples/subsys/usb/webusb/src/main.c

```diff
@@ -24,7 +24,7 @@ static volatile bool data_transmitted;
 static volatile bool data_arrived;
 static u8_t data_buf[64];
 
-/* WebUSB Platform Capability Descriptor */
+/* WebUSB + MS Platform Capability Descriptors */
 static const u8_t webusb_bos_descriptor[] = {
 	/* Binary Object Store descriptor */
 	0x05, 0x0F, 0x1D, 0x00, 0x01,
```

Comment:
Wait wait wait, by reverting the cosmetic changes I also reverted the length and number of following capability descriptors.